### PR TITLE
Prometheus/Explore: Update position of fields in editor

### PR DESCRIPTION
**Pull request by ivanahuckova**
This PR updates the position of fields in Prometheus Query Editor to make it more usable on smaller screens. It also fixes the `Query type` label, that was clickable before. 

**Updated:**
image

**Old:**
image

### CODE REVIEW

**Review by zoltanbedi — APPROVED** [review]
This looks good to me, but I'd like to hear others opinion about this.

File: public/app/plugins/datasource/prometheus/components/PromExploreExtraField.test.tsx [hunk 1]
```diff
@@ -1,28 +1,35 @@
 import React from 'react';
-import { shallow } from 'enzyme';
-import { PromExploreExtraField, PromExploreExtraFieldProps } from './PromExploreExtraField';
+import { render, screen } from '@testing-library/react';
```
Comment (aocenas) [inline public/app/plugins/datasource/prometheus/components/PromExploreExtraField.test.tsx:2]: Dipping into the testing lib, nice :)

**Comment by aocenas** [comment]
So one question is this still ok in dashboards? Not sure if I remember correctly but I think this extra field abstraction was there so it could be changed for dashboards (like not having the step there, not sure). Just wondering if we are not regressing anything by this.

File: public/app/plugins/datasource/prometheus/components/PromQueryField.tsx [hunk 2]
```diff
@@ -331,7 +331,7 @@ class PromQueryField extends React.PureComponent<PromQueryFieldProps, PromQueryF
               {chooserText}
             </ButtonCascader>
           </div>
-          <div className={'gf-form gf-form--grow flex-shrink-1 min-width-15 explore-input-margin'}>
+          <div className={'gf-form gf-form--grow flex-shrink-1 min-width-15'}>
```
[the remark this answers is not in the document]
Comment (ivanahuckova) [reply public/app/plugins/datasource/prometheus/components/PromQueryField.tsx]: Yes, makes sense. Good catch. 👀

**Comment by ivanahuckova** [comment]
@aocenas not sure what exactly do you mean, but this PR doesn't affect the Prometheus dashboard query editor (as you can see on image below, everything looks the same as on master). We are just:
1. refactoring where Radio buttons are stored (moving from `PromExploreQueryEditor.tsx` to `PromExploreExtraField.tsx`)
2. displaying extra field under the query expression input 
3. updating tests, removing redundant snapshot testing
image

So nothing new, just some css changes and moving things around, so it makes more sense.

File: public/app/plugins/datasource/prometheus/components/PromExploreExtraField.test.tsx (hunk 1 above)
Comment (ivanahuckova) [reply public/app/plugins/datasource/prometheus/components/PromExploreExtraField.test.tsx:2]: Hahhaha yeah! I really like react testing library. Moreover, have you seen their logo? 😂 🧡

**Comment by aocenas** [comment]
@ivanahuckova cool in that case, yeah that's what I remembered was that there was difference how the editor looks like in dashboard vs explore but wasn't sure where that was implemented so just wanted to check that.

**Review by aocenas — APPROVED** [review]

**Pull request merged**